app/controllers/overview/jobs/new.js

```diff
@@ -0,0 +1,174 @@
+
+  authenticationEnabled = ['true', 'True', 'TRUE', true].includes(
+    config.harvester.authEnabled
+  );
+
+  @tracked url;
+  @tracked urlValid;
+  @tracked graphName;
+  @tracked graphNameValid;
+  @tracked comment;
+  @tracked selectedJobOperation;
+  @tracked selectedJobOperationValid;
+  @tracked selectedSecurityScheme;
+  @tracked securityScheme = {};
+  @tracked credentials = {};
+
+  @service toaster;
+  @service router;
+  @service store;
+
+  get currentTime() {
+    return new Date();
+  }
+
+  @action
+  updateCredentials(attributeName, credentials) {
+    this.credentials[attributeName] = credentials;
+  }
+  @action
+  updateSecurityScheme(attributeName, securityScheme) {
+    this.securityScheme[attributeName] = securityScheme;
+  }
+
+  @action
+  setSecurityScheme(selected) {
+    this.selectedSecurityScheme = selected;
+  }
+
+  @action
+  setJobOperation(selected) {
+    this.selectedJobOperation = selected;
+  }
+
+  @action
+  validateForm() {
+    //TODO use proper validation library
+    if (this.selectedJobOperation) this.selectedJobOperationValid = true;
+    else this.selectedJobOperationValid = false;
+    if (this.url) this.urlValid = true;
+    else this.urlValid = false;
+    if (this.graphName) this.graphNameValid = true;
+    else this.graphNameValid = false;
+
+    if (this.selectedJobOperation === this.jobImport)
+      return this.selectedJobOperationValid && this.graphNameValid;
+    else return this.selectedJobOperationValid && this.urlValid;
+  }
+
+  @action
+  async cancelCreateAndStartJob() {
+    this.router.transitionTo('overview.jobs');
+  }
+
+  @task
+  *createAndStartJob() {
+    try {
+      if (!this.validateForm()) return;
+
+      const scheduledJob = this.store.createRecord('job', {
+        status: 'http://redpencil.data.gift/id/concept/JobStatus/busy',
+        created: this.currentTime,
+        modified: this.currentTime,
+        creator: this.creator,
+        comment: this.comment,
+        operation: this.selectedJobOperation.uri,
+      });
+      yield scheduledJob.save();
+
+      let dataContainer;
+
+      if (this.selectedJobOperation.uri === this.jobImport) {
+        dataContainer = this.store.createRecord('data-container', {
+          hasGraph: this.graphName,
+        });
+        yield dataContainer.save();
+      } else {
+        const remoteDataObject = this.store.createRecord('remote-data-object', {
+          source: this.url,
+          // This is deliberate, the collector service will set the status and
+          // therefore start the job later:
+          status: undefined,
+          requestHeader:
+            'http://data.lblod.info/request-headers/accept/text/html',
+          created: this.currentTime,
+          modified: this.currentTime,
+          creator: this.creator,
+        });
+        yield remoteDataObject.save();
+
+        const collection = this.store.createRecord('harvesting-collection', {
```

Comment:
No, this might be because the https://github.com/lblod/app-lblod-harvester is quite messy. I suggest checking out the branch add-execute-diff-deletes-service where the feature/login branch has been merged into (even though the feature/login branch has a banner saying that it should not be merged). I have no idea what the goal is for those branches, but for now, I tested with authentication enabled.